toolchain/sem_ir/typed_insts.h

```diff
@@ -1410,6 +1410,15 @@ struct TupleLiteral {
   InstBlockId elements_id;
 };
 
+struct TuplePattern {
```

Comment:
```suggestion
// A tuple pattern, such as `(x, y: i32)`.
struct TuplePattern {
```

Sorry, not sure there's much really to add, but was doing a final glance and realized this would be the only instruction with no comment -- that was an issue in the past with new team members having trouble understanding instructions, so maybe best to have one.